src/pages/auth/Signup.jsx

```diff
@@ -0,0 +1,138 @@
+import React, { useState } from "react";
+import "./auth.css";
+import { authImage } from "../../assets";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuth } from "../../context";
+
+export function Signup() {
+  const defaultFormValue = {
+    email: "",
+    password: "",
+    confirmPwd: "",
+    firstName: "",
+    lastName: "",
+  };
+  const [formData, setFormData] = useState(defaultFormValue);
+  const { token, signupUser } = useAuth();
+  const navigate = useNavigate();
+
+  const signUpHandler = () => {
+    const { email, password, confirmPwd, firstName, lastName } = formData;
+    if (
+      email &&
+      password &&
+      confirmPwd &&
+      firstName &&
+      lastName !== "" &&
```

Comment:
why add this check on lastName only? are you positive that other values will be filled